br/pkg/lightning/backend/local/local.go

```diff
@@ -495,6 +495,18 @@ func NewLocalBackend(
 	return backend.MakeBackend(local), nil
 }
 
+func (local *local) TotalMemoryConsume() int64 {
+	var memConsume int64 = 0
+	local.engines.Range(func(k, v interface{}) bool {
+		e := v.(*Engine)
+		if e != nil {
+			memConsume += e.TotalMemorySize()
+		}
+		return true
+	})
+	return memConsume
```

Comment:
We should also consider the `memConsume` of `local.bufferPool`. When `kvBuffer` is destroyed, its memory is put back to the pool.

Reply:
@gozssky Updated.